package.json

```diff
@@ -42,7 +42,7 @@
     "@floating-ui/react-dom-interactions": "^0.1.1",
     "@fullhuman/postcss-purgecss": "^2.2.0",
     "@heroicons/react": "^1.0.5",
-    "@mdx-js/loader": "1.6.4",
+    "@mdx-js/loader": "^1.6.4",
```

Comment:
```suggestion
    "@mdx-js/loader": "1.6.4",
```

we need to keep it without `^` to make sure dependabot do not update it by default, there were some issues with it